src/components/tabs/PillTabs.js

```diff
@@ -0,0 +1,73 @@
+const React = require('react');
+
+const ButtonGroup = require('../ButtonGroup');
+const Tabbable = require('./Tabbable');
+
+const StyleConstants = require('../../constants/Style');
+
+const PillTabs = Tabbable(({
```

Comment:
Done in c1226e1.

I had to add `*.propTypes = Tabbable.propTypes` to make eslint happy.